### ISSUE TITLE
3P provider logo does not have correct alt text

### ISSUE THREAD

**Issue by spacedmonkey** [GoogleForCreators/web-stories-wp#7274]
## Feature Description

The 3P media provider logo do not have any sort of alt text or screen reader text. Making them completely unaccessible to screen readers. 

Discovered as part of #7272

## Screenshot

![Screenshot 2021-04-21 at 17 14 07](https://user-images.githubusercontent.com/237508/115587219-5435c880-a2c5-11eb-9d38-ea56a083e149.png)
![Screenshot 2021-04-21 at 17 14 12](https://user-images.githubusercontent.com/237508/115587232-5861e600-a2c5-11eb-9302-bff10f9946c5.png)

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
For testing this, verify that the 'powered by' links on the third party media tab get announced

**Comment by csossi** [comment]
Verified in QA
